README.md

```diff
@@ -44,7 +44,26 @@ Create a superuser for admin views
 Run your development server
 
 	$ python manage.py runserver
-	
+
+## Run with composer 
+
+Create your image organilab
```

Comment:
This is for developing proposes, so is not a docker image for production.

Reply:
I have to remove the direction from README.md or just change the title to specify that the following instructions are just for development purposes?

Reply:
Just create an indication that this instructions are for developing purposes.